netdata.spec.in

```diff
@@ -306,6 +312,10 @@ install -m 4750 -p apps.plugin "${RPM_BUILD_ROOT}%{_libexecdir}/%{name}/plugins.
 # Install debugfs.plugin
 install -m 0750 -p debugfs.plugin "${RPM_BUILD_ROOT}%{_libexecdir}/%{name}/plugins.d/debugfs.plugin"
 
+# ###########################################################
+# Install systemd-journal.plugin
+install -m 4750 -p systemd-journal.plugin "${RPM_BUILD_ROOT}%{_libexecdir}/%{name}/plugins.d/systemd-journal.plugin"
```

Comment:
Do we need setuid here?

Reply:
Strictly speaking no, because it gets overridden by the sub-package install syntax below, but this is also how we handle almost every other plugin using capabilities.

Reply:
ok, maybe minor, but it looks like an error that will lead to confusion later. If I check the code I treat "gets overridden by the sub-package install" as a mistake, not expected behavior.